engine/game/petabilities.py

```diff
@@ -77,7 +77,7 @@ def horse_ability(horse: 'PetState', player: 'PlayerState'):
     # Start of combat, gain 0.5L health from the healthiest friend
     def crab_ability(crab: 'PetState', player: 'PlayerState'):
         highest_health = max([pet.get_health() for pet in player.pets if pet != crab and pet is not None])
-        crab.change_health(int(0.5 * highest_health * crab.get_level()))
+        crab._set_health(int(0.5 * highest_health * crab.get_level()))
```

Comment:
Let's make this a public facing method if we're going to use it. (i.e., get rid of the underscore)